kpatch-build/create-diff-object.c

```diff
@@ -922,6 +1020,9 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 				continue;
 			}
 
+#ifdef __powerpc__
+			add_off = 0;
+#else
```

Comment:
Sorry, I don't have a test case in mind.  If we ever get the error then we'll have one :-)